heltour/tournament/models.py

```diff
@@ -1864,27 +1864,39 @@ class Registration(_BaseModel):
                                            null=True)
     weeks_unavailable = models.CharField(blank=True, max_length=255)
 
-    validation_ok = models.BooleanField(blank=True, null=True, default=None)
-    validation_warning = models.BooleanField(default=False)
-    last_validation_try = models.DateTimeField(default=timezone.now)
 
     def __str__(self):
         return "%s" % (self.lichess_username)
 
     def previous_registrations(self):
-        return Registration.objects.filter(lichess_username__iexact=self.lichess_username,
-                                           date_created__lt=self.date_created)
+        return Registration.objects.filter(
+            player=self.player, date_created__lt=self.date_created
+        )
 
     def other_seasons(self):
         return SeasonPlayer.objects.filter(
-            player__lichess_username__iexact=self.lichess_username).exclude(season=self.season)
+            player=self.player).exclude(season=self.season)
 
-    def player(self):
-        return Player.get_or_create(lichess_username=self.lichess_username)
+    @property
+    def lichess_username(self):
+        return self.player.lichess_username
 
     @property
     def rating(self):
-        return self.player().rating_for(league=self.season.league)
+        return self.player.rating_for(league=self.season.league)
+
```

Comment:
We do not need to change this to be able to merge and deploy, but I think we could define an Enum for the validation status, and then have a single validation status function here, which returns the appropriate status, and use the python `match` thing to deal with it when we use it.